I had to add '--root' to the patch for it to not poison the masterdir.
  That part will still be needed as a patch when the next version gets released.
  -- 0x5c
original message below

From a4609318b6fa594f970673d77397f1ca74b4120a Mon Sep 17 00:00:00 2001
From: Morten Linderud <morten@linderud.pw>
Date: Mon, 4 Oct 2021 20:01:53 +0200
Subject: py-smbus/Module.mk: Install with PREFIX defined

If we are building to a defined destdir py-smbus is going to ignore this
and try install to the actual root. This breaks packaging setups as the
install section would fail.

Signed-off-by: Morten Linderud <morten@linderud.pw>
Signed-off-by: Jean Delvare <jdelvare@suse.de>
---
 py-smbus/Module.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/py-smbus/Module.mk b/py-smbus/Module.mk
index 04ceb64..1e2ded1 100644
--- a/py-smbus/Module.mk
+++ b/py-smbus/Module.mk
@@ -22,7 +22,7 @@ clean-python:
 	rm -rf py-smbus/build
 
 install-python:
-	$(DISTUTILS) install
+	$(DISTUTILS) install --prefix="$(PREFIX)" --root="$(DESTDIR)"
 
 all: all-python
 
-- 
cgit 1.2.3-1.el7
